addons/rose/addon/styles/hds/overrides.scss

```diff
@@ -3,3 +3,11 @@
 [class^='hds-form-group'] {
   margin-bottom: 1.5rem;
 }
+/* stylelint-disable selector-class-pattern */
```

Comment:
[selector from hds](https://github.com/hashicorp/design-system/blob/6cf5bff1f50ab2caaaa7f3ba346d353e6c353ff3/packages/components/app/styles/components/form/group.scss#L21) to override the font-weight for boundary. 

<img width="1346" alt="Screen Shot 2023-01-19 at 5 40 54 PM" src="https://user-images.githubusercontent.com/15043878/213599946-7e61e91b-2fcd-4b8d-b030-0204d0c9013b.png">